### PR TITLE
DOC Add session-manager to changelog

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/recipe-cms/issues/47

### CODE REVIEW

File: docs/en/04_Changelogs/4.9.0.md [hunk 1]
```diff
@@ -40,6 +41,12 @@ SilverStripe\Assets\Image:
 
 CMS authors can disable lazy loading on images added via the WYSIWYG editor in the "Edit image" panel by setting the new "Loading" field to "Eager".
 
+### Session manager added to silverstripe/recipe-cms {#session-manager}
+
+The [session manager module](https://github.com/silverstripe/silverstripe-session-manager) is a new security focused module which shows the currently authenticated sessions for the current user. This new module has been added to `silverstripe/recipe-cms` which itself is a requirement of `silverstripe/installer` which is the recommended way to create a new Silverstripe CMS project.
```
Comment (brynwhyman) [inline docs/en/04_Changelogs/4.9.0.md:46]:
```suggestion
The [session manager module](https://github.com/silverstripe/silverstripe-session-manager) is a new security focused feature which allows a CMS user to view and manage their active sessions in the CMS. They can see the device details behind each session and have the ability to revoke these sessions. This new module has been added to `silverstripe/recipe-cms` which is the recommended method of managing Silverstripe CMS dependencies in a project.
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Probably worth mentioning that this thing will work out of the box, but that there's a cronjob that can be enabled with it.

I would also highlight that there's user help site owners can direct their users to.

File: docs/en/04_Changelogs/4.9.0.md (hunk 1 above)
Comment (emteknetnz) [reply docs/en/04_Changelogs/4.9.0.md]: Updated

**Comment by emteknetnz** [comment]
Updated

File: docs/en/04_Changelogs/4.9.0.md [hunk 2]
```diff
@@ -40,6 +41,16 @@ SilverStripe\Assets\Image:
...
+
+Existing projects that have `silverstripe/recipe-cms` as a requirement in `composer.json` will automatically install `silverstripe/session-manager` when they run `composer update`. If your project does not use `silverstripe/recipe-cms` it's recommended that you consider requiring `silverstripe/session-manager` in your project as a security enhancement. The session manager module requires no configuration and works out of the box.
+
+If your site has the [symbiote/silverstripe-queuedjobs](https://github.com/symbiote/silverstripe-queuedjobs) module installed, then a job will automatically be created that will periodically remove old database records created by the session manager module.
+
+There's more information aimed at CMS users available on the [userhelp website](https://userhelp.silverstripe.org/en/4/managing_your_website/session_manager).
```
Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.9.0.md:52]:
```suggestion
CMS users can review the [Session Manager user help](https://userhelp.silverstripe.org/en/4/managing_your_website/session_manager/) for more information on managing their sessions.
```

File: docs/en/04_Changelogs/4.9.0.md [hunk 3]
```diff
@@ -40,6 +41,16 @@ SilverStripe\Assets\Image:
 
 CMS authors can disable lazy loading on images added via the WYSIWYG editor in the "Edit image" panel by setting the new "Loading" field to "Eager".
 
+### Session manager added to silverstripe/recipe-cms {#session-manager}
+
+The [session manager module](https://github.com/silverstripe/silverstripe-session-manager) is a new security focused feature which allows a CMS user to view and manage their active sessions in the CMS within the "My profile" section of the CMS (/admin/myprofile). They can see the device details behind each session and have the ability to revoke these sessions. This new module has been added to `silverstripe/recipe-cms` which is the recommended method of managing Silverstripe CMS dependencies in a project.
+
+Existing projects that have `silverstripe/recipe-cms` as a requirement in `composer.json` will automatically install `silverstripe/session-manager` when they run `composer update`. If your project does not use `silverstripe/recipe-cms` it's recommended that you consider requiring `silverstripe/session-manager` in your project as a security enhancement. The session manager module requires no configuration and works out of the box.
```
Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.9.0.md:48]:
```suggestion
Projects that have `silverstripe/recipe-cms` as a requirement in their `composer.json` will automatically get `silverstripe/session-manager` when you run `composer update`. If your project does not use `silverstripe/recipe-cms`, it's recommended that you require `silverstripe/session-manager` in your composer file as a security enhancement. The session manager module requires no configuration and works out-of-the-box.
```

File: docs/en/04_Changelogs/4.9.0.md [hunk 4]
```diff
@@ -40,6 +41,16 @@ SilverStripe\Assets\Image:
 
 CMS authors can disable lazy loading on images added via the WYSIWYG editor in the "Edit image" panel by setting the new "Loading" field to "Eager".
 
+### Session manager added to silverstripe/recipe-cms {#session-manager}
```
Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.9.0.md:44]: This is a headline feature ... seems like we should have something punchy that sells it a bit more.

```suggestion
### Manage your CMS sessions across devices{#session-manager}
```

File: docs/en/04_Changelogs/4.9.0.md [hunk 5]
```diff
@@ -4,6 +4,7 @@
 
 - [Features and enhancements](#features-and-enhancements)
     - [Image lazy loading](#image-lazy-loading)
+    - [Session manager added to silverstripe/recipe-cms](#session-manager)
```
Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.9.0.md:7]:
```suggestion
    - [Manage your CMS sessions across devices](#session-manager)
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: docs/en/04_Changelogs/4.9.0.md (hunk 2 above)
Comment (emteknetnz) [reply docs/en/04_Changelogs/4.9.0.md:52]: Updated

File: docs/en/04_Changelogs/4.9.0.md (hunk 3 above)
Comment (emteknetnz) [reply docs/en/04_Changelogs/4.9.0.md:48]: Updated

File: docs/en/04_Changelogs/4.9.0.md (hunk 4 above)
Comment (emteknetnz) [reply docs/en/04_Changelogs/4.9.0.md:44]: Updated

File: docs/en/04_Changelogs/4.9.0.md (hunk 5 above)
Comment (emteknetnz) [reply docs/en/04_Changelogs/4.9.0.md:7]: Updated

**Review by maxime-rainville — APPROVED** [review]

**Pull request merged**